projects/cdk/tokens/portal-service.ts

```diff
@@ -0,0 +1,3 @@
+import {InjectionToken} from '@angular/core';
+
+export const TUI_PORTAL_SERVICE = new InjectionToken<{}>('Token for portal service');
```

Comment:
This token is unnecessary, we can just provide the abstract class.

Reply:
deleted